src/utilities/io.cpp

```diff
@@ -372,8 +380,7 @@ void incflo::WritePlotFile()
     if(m_plt_error_w) ncomp += 1;
 #endif
 
-    // Error in nodal pressure (computed vs exact)
-    if(m_plt_error_p) ncomp += 1;
+    // Error in nodal pressure (computed vs exact) if(m_plt_error_p) ncomp += 1;
```

Comment:
Did you mean to comment out the option m_plt_error_p? If so, maybe an Abort should be added if the inputs file tries to set this?